app/workers/transactional_email_analytics_job.rb

```diff
@@ -36,6 +36,19 @@ def perform
     end
   end
 
+  # constantize all the mailers so TransactionalEmailMailer.descendants has something
+  def self.all_mailers
```

Comment:
I think https://github.com/department-of-veterans-affairs/vets-api/blob/56ecc229ba6ec8f51e460dc50bfa12ce6eebcb8e/app/workers/transactional_email_analytics_job.rb#L45 should use `all_mailers` instead of `descendants`.  When I suggested this approach it was motivated by getting away from using `descendants`.